routes/index.js

```diff
@@ -1,20 +1,34 @@
 import express from 'express'
 import passport from 'passport'
 const router = express.Router()
+import { getRandomlyPlaylist } from '../controllers/playlistController'
+import { getSongById, updateSongPlays } from '../controllers/songController'
+import { getAlbumById, getAlbums } from '../controllers/albumController'
 import {
   validateSignup,
   signUp,
   signIn,
   getCurrentUser,
   requestReset,
   resetPassword,
-  fbSignIn
+  fbSignIn,
 } from '../controllers/authContorller'
 
 const requireAuth = passport.authenticate('jwt', { session: false })
 const requireSignin = passport.authenticate('local', { session: false })
 const facebookSignin = passport.authenticate('facebook', { session: false })
 
+// playlist
+router.get('/api/playlist/randomly', getRandomlyPlaylist)
+
+// songs
+router.get('/api/song', getSongById)
```

Comment:
In these routes, I would include id parameter as part of the url, not query string, to increase readability.

router.get('/api/song/:song_id', getSongById);
router.post('/api/song/:song_id/plays', updateSongPlays);
etc.